src/components/calcite-tooltip/calcite-tooltip.stories.ts

```diff
@@ -37,23 +39,29 @@ export default {
   }
 };
 
-export const Simple = (): string => {
-  return `
-      <div>
-        ${referenceElementHTML}
-        <calcite-tooltip
-        class="calcite-theme-light"
-          reference-element="reference-element"
-          placement="${select("placement", calcite_placements, "auto")}"
-          offset-distance="${number("offset-distance", 6)}"
-          offset-skidding="${number("offset-skidding", 0)}"
-          ${boolean("open", true)}
-        >
-          ${contentHTML}
-        </calcite-tooltip>
-      </div>
-    `;
-};
+export const Simple = (): string => html`
+  <div>
+    ${referenceElementHTML}
+    <calcite-tooltip
+      class="calcite-theme-light"
+      reference-element="reference-element"
+      placement="${select("placement", calcite_placements, "auto")}"
+      offset-distance="${number("offset-distance", 6)}"
+      offset-skidding="${number("offset-skidding", 0)}"
+      ${boolean("open", true)}
+    >
+      ${contentHTML}
+    </calcite-tooltip>
+  </div>
+`;
+
+Simple.decorators = [
```

Comment:
Good news, this works. Maybe we can create a utility to handle this because it seems the `typings` for Story aren't there and creating the fragment isn't very pretty. We can just have a utility that we pass the steps and it handles setting the decorators on the story, What do you think @jcfranco and @eriklharper?

Reply:
Nice! 

Your suggestion seems doable. Any idea on what the API & usage would look like?

Reply:
Awesome dude, thanks for following up on Loyal's response.

Reply:
We'll definitely want to write a utility to make doing the steps easier and reduce boilerplate.